docs/core/diagnostics/dotnet-trace.md

```diff
@@ -259,6 +275,205 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
+
+  Example: `--perf-event-tracepoints syscalls:sys_enter_execve,sched:sched_switch,sched:sched_wakeup`
+
+- **`--profiles <list-of-comma-separated-profile-names>`**
+
+  A named, pre-defined set of provider configurations for common tracing scenarios. You can specify multiple profiles as a comma-separated list. When multiple profiles are specified, the providers and settings are combined (union), and duplicates are ignored.
+
+  Available profiles:
+
+  | Profile | Description |
+  |---------|-------------|
+  |`dotnet-common`|Lightweight .NET runtime diagnostics (GC, JIT, loader/assembly loader, threading, contention). Designed to stay low overhead.|
+  |`dotnet-thread-time`|Samples .NET thread stacks (~100 Hz) to identify hotspots over time. Uses the runtime sample profiler with managed stacks.|
+  |`kernel-cpu`|Kernel CPU sampling (perf-based), emitted as `Universal.Events/cpu`, for precise on-CPU attribution.|
+  |`kernel-cswitch`|Kernel thread context switches, emitted as `Universal.Events/cswitch`, for on/off-CPU and scheduler analysis.|
+  |`kernel-precise-thread-time`|Combines `kernel-cpu` + `kernel-cswitch` for a precise thread-time view using kernel events only.|
+  |`gc-verbose`|Tracks GC collections and samples object allocations.|
+  |`gc-collect`|Tracks GC collections only at very low overhead.|
+
+  > [!NOTE]
+  > The former default `cpu-sampling` profile is now named as `dotnet-thread-time`.
+
+#### Trace Collection Options
+
+- **`--format {Chromium|NetTrace|Speedscope}`**
+
+  Sets the output format for the trace file conversion. The default is `NetTrace`.
+
+- **`-o|--output <trace-file-path>`**
+
+  The output path for the collected trace data. If not specified it defaults to `<appname>_<yyyyMMdd>_<HHmmss>.nettrace`, for example, `myapp_20210315_111514.nettrace``.
+
+- **`--duration <time-to-run>`**
+
+  The time for the trace to run. Use the `dd:hh:mm:ss` format. For example `00:00:00:05` will run it for 5 seconds.
+
+#### .NET Process Target Options
+
+See [Default collection behavior](#default-collection-behavior)
+
+- **`-n, --name <name>`**
+
+  The name of the process to collect the trace from.
+
+  > [!NOTE]
+  > Should the target application have a `TMPDIR` enviornment variable set, `dotnet-trace` will need to share the same `TMPDIR` value, otherwise events will not be collected from the target application.
+
+- **`-p|--process-id <PID>`**
+
+  The process ID to collect the trace from.
+
+  > [!NOTE]
+  > Should the target application have a `TMPDIR` enviornment variable set, `dotnet-trace` will need to share the same `TMPDIR` value, otherwise events will not be collected from the target application.
+
+- **`-- <command>`**
```

Comment:
This may not be trivial to implement. I think its fine if we don't have it. For the collect verb this was important because it wasn't easy to trace a process from startup otherwise. With collect-linux being process-wide by default it shouldn't be hard for users to start a trace, then start their process of interest.

Reply:
Removed from collect-linux